textx/scoping/__init__.py

```diff
@@ -266,3 +268,22 @@ def get_all_models_including_attached_models(model):
     else:
         models = [model]
     return models
+
+
+def is_file_included_by_model(filename, model):
```

Comment:
Maybe it would be better to have `is_file_included` on `model` object?

Reply:
I like the idea. Where do you think is a good place to add the method. We do not have a "class Model" or so... Where do you suggest to add those methods (or attributes).

Reply:
I was thinking about adding them like we add special attributes on the constructed model but after second thought I don't think that would be a good idea. It would pollute the namespace of user model object and adding them with `_tx_` prefix would be plain ugly.

Reply:
I just had the same idea. I will keep free functions instead.

Reply:
Done